distributed/client.py

```diff
@@ -1367,6 +1373,16 @@ def wait_for_workers(self, n_workers=0, timeout=None):
             Time in seconds after which to raise a
             ``dask.distributed.TimeoutError``
         """
+        if n_workers is no_default:
+            warnings.warn(
+                "Client.wait_for_workers requires passing the argument `n_workers`. Not passing any argument will no longer be supported in future versions.",
```

Comment:
Minor clarification suggestion. Feel free to include/ignore as you see fit 

```suggestion
                "Please specify the `n_workers` argument when using `Client.wait_for_workers`. Not specifying `n_workers` will no longer be supported in future versions.",
```